packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.base.tsx

```diff
@@ -236,9 +237,10 @@ export class TooltipHostBase extends React.Component<ITooltipHostProps, ITooltip
 
   private _toggleTooltip = (isTooltipVisible: boolean): void => {
     if (this.state.isTooltipVisible !== isTooltipVisible) {
+      this._defaultTooltipId = getId('tooltip');
```

Comment:
is this redundant with line 40?

Reply:
No, it's meant to re-initialize the ID since it's a different tooltip, meant to preserve the previous behavior of having a different ID each time the tooltip opens......but that may not actually be necessary/desirable, so I can remove it.